extensions/wikia/ImageReview/ImageReviewEventsHooks.class.php

```diff
@@ -1,11 +1,19 @@
 <?php
 
-use \Wikia\Logger\WikiaLogger;
-use \Wikia\Tasks\Tasks\ImageReviewTask;
+use Wikia\Logger\WikiaLogger;
+use Wikia\Rabbit\ConnectionBase;
+use Wikia\Tasks\Tasks\ImageReviewTask;
 
 class ImageReviewEventsHooks {
 	public static function onUploadComplete( UploadBase $form ) {
-		static::createAddTask( $form->getTitle() );
+		global $wgCityId, $wgImageReviewTestCommunities;
+
+		// TODO: community switch
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)